validate.py

```diff
@@ -127,10 +128,7 @@ def determine_if_mergeable():
       else:
         rejections.append(user)
 
-  if rejections:
-    raise Exception('Rejected by: %s' % (' '.join(rejections)))
```

Comment:
I object to removing the reject-as-veto functionality in this PR.

(I'm not against it *in principle*, but I don't like seeing it as part of this rule change.)